### DISCUSSION TITLE
Parry skill

### DISCUSSION THREAD

**Discussion by wwwwwwwwwwiiiiiiiiii** [apokryphus/advanced-combat-system#19] [Q&A]
I really like parry skill moveset in this mod. It makes the game's moves more diverse. But one thing I don't like is the moveset W + parry . It alternates between shoulder bash and sidestep slash. For me, this reduces the consistency in the use of stances in combat. How can I change this? for example I want to set the shoulder bash to S + parry and the siderstep slash to be W + parry.

**Comment by apokryphus** [comment]
This is to prevent the player from spamming shoulderbash repeatedly, since the stagger effect is way too strong in this game. 

However, W + doubletap parry will always perform a kick unless the player has any throwing knives from ACS in the inventory, which is basically the same as shoulder bash.

Alternatively you can just cancel out of the animation with an additional input.

**Reply by wwwwwwwwwwiiiiiiiiii** [reply]
no, I mean I want to remove the shoulderbash. Sorry for not making it clear

**Reply by apokryphus** [reply]
Ah I see. I'll consider to make it adjustable, but the same thing applies, as you can just cancel out of the animation.
